manager/controllers/app/blueprint_controller.go

```diff
@@ -327,8 +329,18 @@ func NewBlueprintReconciler(mgr ctrl.Manager, name string, helmer helm.Interface
 
 // SetupWithManager registers Blueprint controller
 func (r *BlueprintReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// 'UpdateFunc' and 'CreateFunc' used to judge if the event came from within the blueprint's namespace.
+	// If that is true, the event will be processed by the reconciler.
+	// If it's not then it is a rogue event created by someone outside of the control plane.
+	p := predicate.Funcs{
+		CreateFunc: func(e event.CreateEvent) bool {
```

Comment:
Is CreateFunc enough? (i.e., no need for UpdateFunc)

Reply:
@roee88 I originally included Update as well, but removed it since there is no way to update something that cannot be created.  Right?

Reply:
@roee88 I added Update